### PR TITLE
JmeSurfaceView: Javadocs format improves

### PR DESCRIPTION
**Pull request by Scrappers-glitch**
Some code format and javadocs fixes

### CODE REVIEW

**Comment by Scrappers-glitch** [comment]
Let me know if you find something wrong with this PR..

Notice : The order of the internal classes have been changed by the code reformater, they are now at the end of the class....

File: jme3-android/src/main/java/com/jme3/app/jmeSurfaceView/JmeSurfaceView.java [hunk 1]
```diff
@@ -946,36 +882,136 @@ public void setOnRendererStarted(OnRendererStarted onRendererStarted) {
...
+     * Determines whether the app context would be destructed
+     * with the holder activity context in case of {@link DestructionPolicy#DESTROY_WHEN_FINISH} or be
+     * spared for a second use in case of {@link DestructionPolicy#KEEP_WHEN_FINISH}.
+     * Default value is : {@link DestructionPolicy#DESTROY_WHEN_FINISH}.
+     *
+     * @see JmeSurfaceView#setDestructionPolicy(DestructionPolicy)
      */
-    public boolean isShowErrorDialog() {
-        return showErrorDialog;
+    public enum DestructionPolicy {
```
Comment (Scrappers-glitch) [inline jme3-android/src/main/java/com/jme3/app/jmeSurfaceView/JmeSurfaceView.java:930]: Due to code re-formating, those are moved to the bottom of the class, so the class constructor are always at top as java conventions states : https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html#1852

**Comment by stephengold** [comment]
I notice you added blank lines to organize imports into groups. Section 3.3.3 of the style guide says:
> If there are both static and non-static imports, a single blank line separates the two blocks. There are no other blank lines between import statements.

**Comment by Scrappers-glitch** [comment]
> I notice you added blank lines to organize imports into groups. Section 3.3.3 of the style guide says:
> 
> > If there are both static and non-static imports, a single blank line separates the two blocks. There are no other blank lines between import statements.

The IDEA code reformater does this, it separates api import statements into [android, jme, java]. Should i revert this ?

**Comment by stephengold** [comment]
> Should i revert this ?
If you can.

**Comment by Scrappers-glitch** [comment]
Done, let me know if there are other changes.

**Comment by stephengold** [comment]
Note:  there are 2 PRs on the same branch. It appears that, as a consequence, the blank lines will be added in #1805 and then removed by #1808.

**Comment by stephengold** [comment]
I was wrong about the 2 PRs on the same branch. I'm glad that worked out cleanly.

@Scrappers-glitch Thank you for your latest contribution to the Engine!

**Pull request merged**